DiamondLightSource/python-artemis
# Mxgda 3764: Run plan from flask server

See https://jira.diamond.ac.uk/browse/MXGDA-3764

To test:
* Update dependencies
* Run modified tests and confirm they pass
* Start up a runner as in README instructions and start/stop some scans
* Confirm runner acts as expected and starts a scan based on the ophyd/bluesky logs in /tmp/
